docker/test/docker-compose.yml

```diff
@@ -10,8 +10,8 @@ services:
     ports:
       - 8080:8080
     volumes:
-      - ./logs/:/opt/orbs/logs
-      - ./e2e-config/:/opt/orbs/config
+      - ../../logs/:/opt/orbs/logs
```

Comment:
let's find a better place for `./logs` in the project root, maybe `./_logs` or `./_output/logs`

The idea is to indicate that this folder does not contain go source files (it looks like the go package logs)

Also, if we have more types of output files - it's better they're all in the same directory and not scattered in the root level of the project

Reply:
Moved to `_logs`.